### PR TITLE
Feat: integration page implements

### PR DESCRIPTION
**Pull request by wangbow**
### Description of your changes

Fixes # integration page implements

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary.
- [x] Run `yarn lint` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: src/locals/Zh/zh.json [hunk 1]
```diff
@@ -483,8 +483,11 @@
   "Sign in": "登陆",
   "Logout": "退出登陆",
   "Please input the password": "请输入密码",
-  "Please input the username": "请输入用户账号",
+  "Please input the username or email": "请输入用户账号或邮箱",
```
Comment (barnettZQG) [inline src/locals/Zh/zh.json:486]:
```suggestion
  "Please input the username": "请输入用户账号",
```

File: src/locals/Zh/zh.json [hunk 2]
```diff
@@ -483,8 +483,11 @@
...
   "The password should be 8-16 bits and contain at least one number and one letter": "密码必须包含至少一个数字和字母，长度在8-16位之间",
+  "IntegrationsList": "集成列表",
```
Comment (barnettZQG) [inline src/locals/Zh/zh.json:488]:
```suggestion
  "Integrations": "集成列表",
```

File: src/model/integrations.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { getConfigTypes } from '../api/integration';
+import type { IntegrationBase } from '../interface/integrations';
+
+const integrations: any = {
+  namespace: 'integrations',
+  state: {
+    integrationsConfigTypes: [],
+  },
+  reducers: {
+    updateIntegrationListType(state: IntegrationBase, { payload }: { payload: IntegrationBase }) {
```
Comment (barnettZQG) [inline src/model/integrations.ts:10]:
```suggestion
    updateIntegrationType(state: IntegrationBase, { payload }: { payload: IntegrationBase }) {
```

File: src/layout/LeftMenu/menu.js [hunk 4]
```diff
@@ -92,6 +94,13 @@ export function getLeftSlider(pathname) {
           navName: 'Projects',
           permission: { resource: 'project:*', action: 'list' },
         },
+        {
+          className: isIntegration,
+          link: '/integrations',
+          iconType: 'indent',
+          navName: 'IntegrationsList',
```
Comment (barnettZQG) [inline src/layout/LeftMenu/menu.js:101]: Integrations
```suggestion
          navName: 'Integrations',
```

File: src/pages/Integrations/components/CreateIntegrationDialog/index.tsx [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+                    }}
+                  />
+                </FormItem>
+              </Col>
+            </Row>
+
+            <Row>
+              <Col span={24} style={{ padding: '0 8px' }}>
+                <Group
+                  title={i18n.t('Component Properties')}
```
Comment (barnettZQG) [inline src/pages/Integrations/components/CreateIntegrationDialog/index.tsx:273]:
```suggestion
                  title={i18n.t('Config Properties')}
```

File: src/pages/Integrations/index.tsx [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+        <div className="create-btn">
+          <Button type="primary" onClick={this.handleClickCreate}>
+            <Translation>New</Translation>
+          </Button>
+        </div>
+        <Table locale={locale.Table} dataSource={list} hasBorder={false} loading={isLoading}>
+          {columns && columns.map((col, key) => <Column {...col} key={key} align={'left'} />)}
+        </Table>
+
+        <Pagination
```
Comment (barnettZQG) [inline src/pages/Integrations/index.tsx:224]: The API does not support pagination.

File: src/locals/Zh/zh.json (hunk 1 above)
Comment (wangbow) [reply src/locals/Zh/zh.json]: done

File: src/locals/Zh/zh.json (hunk 2 above)
Comment (wangbow) [reply src/locals/Zh/zh.json]: done

File: src/model/integrations.ts (hunk 3 above)
Comment (wangbow) [reply src/model/integrations.ts]: done

File: src/layout/LeftMenu/menu.js (hunk 4 above)
Comment (wangbow) [reply src/layout/LeftMenu/menu.js]: done

File: src/pages/Integrations/components/CreateIntegrationDialog/index.tsx (hunk 5 above)
Comment (wangbow) [reply src/pages/Integrations/components/CreateIntegrationDialog/index.tsx]: done

**Review by barnettZQG — APPROVED** [review]
LGTM

**Pull request merged**